src/tests/system/tests/test_sssctl.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
+
+import pytest
+
+from lib.sssd.roles.client import Client
+from lib.sssd.topology import KnownTopology
+
+
+@pytest.mark.tier(0)
+@pytest.mark.ticket(bz=2100789)
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_sssctl__check_id_provider(client: Client):
+    """
+    :title: Check id_provider in domain section with sssctl config-check command
+    :setup:
+        1. Create the sssd.conf, here we are using provider as a LDAP server
+    :steps:
+        1. Remove id_provider from domain section.
+        2. Check error message using sssctl config-check.
+    :expectedresults:
+        1. Successfully remove id_provider from domain section.
+        2. Successfully get the error message.
+    :customerscenario: False
+    :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2100789
+    """
+    # start the sssd, with deafult configuration with a LDAP server.
+    client.sssd.start()
```

Comment:
There is actually no need for SSSD to be running to check the configuration.

Reply:
Yes, but `client.sssd.start()` actually creating sssd.conf file and the start with our define topology, here I decide to have LDAP, so it will create the sssd.conf considering LDAP as a server and the start it with default config.

Reply:
I understand. It's OK then.